pkgs/applications/audio/cdparanoia/default.nix

```diff
@@ -1,10 +1,11 @@
 { lib, stdenv, fetchurl, autoreconfHook, gnu-config, IOKit, Carbon }:
 
 stdenv.mkDerivation rec {
-  name = "cdparanoia-III-10.2";
+  pname = "cdparanoia-III";
+  version = "10.2";
 
   src = fetchurl {
-    url = "http://downloads.xiph.org/releases/cdparanoia/${name}.src.tgz";
+    url = "https://downloads.xiph.org/releases/${pname}/${pname}-${version}.src.tgz";
```

Comment:
The directory name does not exist.

Reply:
https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz WOFM

Reply:
This link resolves to `downloads.xiph.org/releases/cdparanoia-III/cdparanoia-III-10.2.src.tgz`, though.